assertions/runner.go

```diff
@@ -37,9 +36,9 @@ func Parallell(ctx context.Context, t Testable) error {
 				if err := asserter.Assert(res, args); err != nil {
 					resp, _ := httputil.DumpResponse(res, true)
 					req, _ := httputil.DumpRequest(res.Request, true)
-					return fmt.Errorf("%v : %s : %s : %s", ctx.Value(t), err, string(req), string(resp))
+					return fmt.Errorf("------> %v :  %s : FAILED : %s \n Request:\n %s \nResponse:\n %s", ctx.Value(t), asrt, err, string(req), string(resp))
```

Comment:
error format should follow [go's code review guidelines](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)